HacktoberfestProject.Web/wwwroot/css/site.css

```diff
@@ -119,4 +119,6 @@ body {
 	animation-play-state: paused;
 }
 
-
+.dotNetFoundation {
```

Comment:
You might want to consider using naming similar to other classes. This way, all of the the code looks similar, making it easier for others to understand.

In this case, it would be something like `dotnet-foundation` instead of the camelCasing `dotNetFoundation`.

Reply:
Reject this pull request, and i'll change it. Thanks for the feedback.

Reply:
Oh, we don't have to reject it! If you push the change to the branch that this PR originates from (`screechrat:css-updates`), the change will be added to the PR. And then we can approve and merge the PR.